src/platforms/php/guides/laravel/configuration/laravel-options.mdx

```diff
@@ -49,3 +103,31 @@ In Laravel this is `App\Http\Controllers` by default but you can override this u
 ```php {filename:config/sentry.php}
 'controllers_base_namespace' => env('SENTRY_CONTROLLERS_BASE_NAMESPACE', 'App\\Http\\Controllers'),
 ```
+
+## Closures and config caching
+
+Sometimes the SDK requires a closure as an option value. The causes problems when using `php artisan config:cache` resulting in the `Your configuration files are not serializable` error.
+
+We can work around that by providing a callable instead of a closure. In this example we are using the `traces_sampler` option but this can be used for any other option that accepts a closure:
```

Comment:
```suggestion
We can work around that by providing a callable instead of a closure. In this example we are using the `traces_sampler` option, but this can be used for any other option that accepts a closure:
```